gravitee-apim-console-webui/src/management/settings/api-portal-header/migrated/api-portal-header.component.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, OnDestroy, OnInit } from "@angular/core";
+import { EMPTY, Subject } from "rxjs";
+import { catchError, filter, switchMap, takeUntil, tap } from "rxjs/operators";
+import {
+  GioConfirmDialogComponent,
+  GioConfirmDialogData
+} from "@gravitee/ui-particles-angular";
+import { MatDialog } from "@angular/material/dialog";
+
+import { DialogData, HeaderDialogComponent } from "./header-dialog/header-dialog.component";
+
+import { EnvironmentApiHeadersService } from "../../../../services-ngx/environment-api-headers.service";
+import { ApiPortalHeader } from "../../../../entities/apiPortalHeader";
+import { SnackBarService } from "../../../../services-ngx/snack-bar.service";
+
+
+@Component({
+  selector: "app-api-portal-header",
+  templateUrl: "./api-portal-header.component.html",
+  styleUrls: ["./api-portal-header.component.scss"],
+  host: {
+    class: "bootstrap gv-sub-content"
```

Comment:
you can delete this line. 
the bootstrap class is useful for components that have not been migrated so that this css applies. here only material should be used.

Reply:
removed